src/views/NewDinners/DinnerForm.tsx

```diff
@@ -36,7 +36,7 @@ export const DinnerForm = ({ existingDinner }: Props) => {
     console.log(values);
   }
   const tags = form.watch("tags");
-  const newTag = form.watch("newTag");
+  const newTag = form.watch("newTag")?.trim();
```

Comment:
What does this do?

Reply:
We had an issue where we had some duplicate tags where the only difference was a trailing space. "Vegetables" and "Vetegables " for instance. This takes care of that. 

I also went through the db and fixed the existing duplicates.